### PR TITLE
Static analysis settings

### PR DESCRIPTION
**Pull request by katulino**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Unit Tests
- [ ] Attached examples
- [ ] Own example attached to Pull Request

**Test Configuration**:
* RFEM / RSTAB / RSECTION version:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: includes/AnalysisSettings/StaticAnalysisSettings.js [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+// Main function
+function StaticAnalysisSettings(no,
+                                analysisType,
+                                equationSolver,
+                                nonlinearMethod,
+                                comment,
+                                params)
+{
+
+   /**
```
Comment (jarabroz) [inline includes/AnalysisSettings/StaticAnalysisSettings.js:85]: Please update DocStrings

File: includes/AnalysisSettings/StaticAnalysisSettings.js [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+    console.log("Wrong equation solver input. Value was: " + solverType);
+    console.log("Correct values are: ( " + Object.keys(EquationSolver_dict) + ")");
+    equationSolver = "METHOD_OF_EQUATION_SYSTEM_DIRECT";
+  };
+  return equationSolver 
+};
+
+
+// Main function
+function StaticAnalysisSettings(no,
```
Comment (jarabroz) [inline includes/AnalysisSettings/StaticAnalysisSettings.js:77]: Would it be possible to create separate prototype methods based on analysis type - Geometrically linear, Second Order and Large deformation. Then each function has its own argument which could be clear to user.

File: includes/AnalysisSettings/StaticAnalysisSettings.js [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+    "Picard"             : "PICARD",
+    "Combined"           : "NEWTON_RAPHSON_COMBINED_WITH_PICARD",
+    "Postcritical"       : "NEWTON_RAPHSON_WITH_POSTCRITICAL_ANALYSIS",
+    "Newton"             : "NEWTON_RAPHSON",
+    "Constant stiffness" : "NEWTON_RAPHSON_WITH_CONSTANT_STIFFNESS",
+    "Dynamic"            : "DYNAMIC_RELAXATION",
+  };
+
+  const nonlinearMehodsSwitcher = {
+    "Second-order (P-Δ)"   : nonlinearMethods_secondOrder_dict,
```
Comment (jarabroz) [inline includes/AnalysisSettings/StaticAnalysisSettings.js:38]: this (P-Δ) could be potentially difficult to write on standard keyboard.

Comment (jarabroz) [reply includes/AnalysisSettings/StaticAnalysisSettings.js:38]: David is using xx_yy if there is two-words argument see https://github.com/Dlubal-Software/Dlubal_JavaScript_Library/blob/71c34948bbec21d28ae6d98294e9279cfd4f21a1/includes/TypesForMembers/MemberEccentricity.js#L160

Comment (katulino) [reply includes/AnalysisSettings/StaticAnalysisSettings.js:38]: > 

This parameter is inside FREM is has no interaction with user

File: includes/AnalysisSettings/StaticAnalysisSettings.js (hunk 2 above)
Comment (katulino) [reply includes/AnalysisSettings/StaticAnalysisSettings.js:77]: New prototypes created

**Pull request merged**